assets/airports/kdab.json

```diff
@@ -0,0 +1,6847 @@
+{
+    "airac": 2201,
+    "radio": {
+        "twr": "Daytona Tower",
+        "app": "Daytona Departure",
+        "dep": "Daytona Approach"
+    },
+    "icao": "KDAB",
+    "iata": "DAB",
+    "magnetic_north": -5,
+    "ctr_radius": 80,
+    "ctr_ceiling": 10000,
+    "initial_alt": 4000,
+    "position": [29.1799089, -81.0580422, "34ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": [29.1799089, -81.0580422],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 40,
+        "speed": 9
+    },
+    "arrivalRunway": "7L",
+    "departureRunway": "7L",
+    "defaultMaps": ["East Operations"],
+    "airspace": [
```

Comment:
According to the diagrams in the ZJX SOP, the airspace is a little different and should look like this:
![image](https://user-images.githubusercontent.com/13125311/152651049-bba97b4a-a6c4-4b21-a23a-c42299205982.png)